### PR TITLE
Feature create course

### PR DESCRIPTION
**Pull request by naberken**
I need help fixing the courses.js tests. But the code is functional and I know people are waiting on it, so maybe we pull it in to staging now.

### CODE REVIEW

**Comment by EndureBlackout** [comment]
It will not let us merge until your tests are done. If you need help let me know we can go over them some time. @naberken

**Comment by naberken** [comment]
Yeah that would be great, we can talk about it more during the meeting today

**Review by EndureBlackout — APPROVED** [review]

**Review by MattFriemoth — CHANGES_REQUESTED** [review]
Nice

**Comment by MattFriemoth** [comment]
Since this pull request is now old, please attempt to merge the new staging branch with this one

File: routes/courses.js [hunk 1]
```diff
@@ -1,13 +1,59 @@
...
+      return res.send(course);
+    } catch (error) {
+      next(error);
+    }
+  });
+
+  // needs the id of the user making the request for authorization
+  router.post('/', authorizeSession, async (req, res, next) => {
+    try {
+      const userId = res.locals.userId;
```
Comment (EndureBlackout) [inline routes/courses.js:32]: @MattFriemoth the authentication is right here. Appears correct to me?

Comment (MattFriemoth) [reply routes/courses.js:32]: Oh! You are correct. My mistake. Should have looked closer.

**Review by MattFriemoth — APPROVED** [review]

**Pull request merged**